### DISCUSSION TITLE
Congress of the Federated States of Micronesia

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#65] [Show and tell]
### Channel ID

FSMCongress.fm

### Stream URL

https://youtu.be/pqJXGeeKbkI

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by fotvnetwork** [comment]
https://www.youtube.com/c/V6AHPOHNPEI/live
